### ISSUE TITLE
systemd unit file

### ISSUE THREAD

**Issue by tim-seoss** [static-web-server/static-web-server#35]
Hello,

I thought it might be good to supply an example systemd unit file.  Here's a WiP:

```
[Unit]
Description=Static Web Server
After=network.target network-online.target

[Service]
Type=simple
EnvironmentFile=/etc/default/static-web-server
ExecStart=/usr/local/bin/static-web-server
Restart=on-abort

DynamicUser=true
SupplementaryGroups=www-data

NoNewPrivileges=yes
PrivateTmp=yes
ProtectSystem=strict
ProtectHome=yes

CapabilityBoundingSet=
RestrictNamespaces=true
RestrictAddressFamilies=AF_INET AF_INET6
PrivateDevices=true
PrivateUsers=true
PrivateTmp=true
ProtectClock=true
ProtectControlGroups=true
ProtectKernelLogs=true
ProtectKernelModules=true
ProtectKernelTunables=true
ProtectProc=invisible
ProcSubset=pid
RestrictSUIDSGID=true
SystemCallArchitectures=native
RestrictRealtime=true
LockPersonality=true
RemoveIPC=true
MemoryDenyWriteExecute=true
UMask=007
ProtectHostname=true
# The following could probably be restricted further, with care.
#e.g. "SystemCallFilter=@basic-io @network-io @process @file-system"
#... would be a good starting point.
SystemCallFilter=@system-service

[Install]
WantedBy=multi-user.target
```

**Comment by joseluisq** [comment]
Sure, happy to receive a PR.
Maybe an `examples` dir for it.

**Comment by tim-seoss** [comment]
I'll see if it's possible to lock it down a bit further, and create a PR.

**Comment by joseluisq** [comment]
@tim-seoss should this be resolved by #40?

**Comment by tim-seoss** [comment]
Yes, sorry my oversight - resolved by #40.
